src/cloud/live/go-live-checklist.md

```diff
@@ -69,7 +69,7 @@ The following are recommended changes and checks:
 
 ## Performance testing {#performance}
 
-We recommend that you review the [Magento Performance Toolkit]({{ site.mage2bloburl }}/setup/performance-toolkit) options as part of your pre-launch readiness process.
+We recommend that you review the [Magento Performance Toolkit](https://github.com/magento/magento2/tree/2.3/setup/performance-toolkit) options as part of your pre-launch readiness process.
```

Comment:
use `{{ site.mage2bloburl }}/{{ site.version }}` instead of `https://github.com/magento/magento2/tree/2.3` to always point to the mainline version.